### DISCUSSION TITLE
What inspired the choice to use scheme\lisp\s-expressions for the queries?

### DISCUSSION THREAD

**Discussion by jaidetree** [tree-sitter/tree-sitter#1896] [Q&A] [answered]
I love lisp languages myself so was surprised to find tree-sitter used scheme syntax to define the queries.

What inspired the choice?

**Comment by ahlinc** [comment, marked as answer]
@eccentric-j You can find more information in the following old PR: https://github.com/tree-sitter/tree-sitter/pull/444
